psychopy/app/pavlovia_ui/_base.py

```diff
@@ -89,7 +89,7 @@ def logout(self):
     def login(self):
         self._loggingIn = True
         authURL, state = pavlovia.getAuthURL()
-        self.browser.Bind(wx.html2.EVT_WEBVIEW_LOADED, self.checkForLoginURL)
```

Comment:
I'm pretty sure that removing this line will break successful login when the page *does* load

Reply:
I suggest binding EVT_WEBVIEW_ERROR to another method, like `onConnectionErr()` rather than sending it to `checkForLoginURL`? I presume when there is an error there's no point in checking the login url

Reply:
Ok, I was only suggesting that something happens when there is no internet connection and the user attempts to log in. My first idea was to load a webpage from resources that informs the user an internet connection is required or that there is some issue causing the login URL to fail. The use of ```EVT_WEBVIEW_ERROR``` allows us to check whether an error occurs during the loading of the login screen, and so catching an error in ```checkForLoginURL``` and ending the dlg on unsuccessful attempts to reach pavlovia seemed like a good idea.  Yes, we can create a new method called ```onConnectionErr``` which calls ```checkForLoginURL``` on successful login attempts, or we could revert back to the original arrangement and find a better solution.